app/src/server/emails/sendEmail.ts

```diff
@@ -0,0 +1,34 @@
+import * as SibApiV3Sdk from "@sendinblue/client";
```

Comment:
I probably would have used a generic SMTP email sender for this instead of using Brevo's API directly. That way it's easy for us to swap out for another provider in local dev for testing.

Eg. this is how I set up emails with Brevo in another project:

```typescript
import { marked } from "marked";
import nodemailer from "nodemailer";
import { env } from "~/env.mjs";

const transporter = nodemailer.createTransport({
  // All the SMTP_ env vars come from https://app.brevo.com/settings/keys/smtp
  // @ts-expect-error nodemailer types are wrong
  host: env.SMTP_HOST,
  port: env.SMTP_PORT,
  auth: {
    user: env.SMTP_LOGIN,
    pass: env.SMTP_PASSWORD,
  },
});

export const sendEmail = async (options: { to: string; subject: string; body: string }) => {
  const bodyHtml = marked.parseInline(options.body, { mangle: false });

  return await transporter.sendMail({
    from: "noreply@translato.ai",
    to: options.to,
    subject: options.subject,
    html: bodyHtml,
    text: options.body,
  });
};
```

Reply:
Good idea